### PR TITLE
Add decision made on versioned data

### PR DESCRIPTION
**Pull request by jcoyne**
We decided to use a custom layout that stores content addressed files and creates symbolic links to preserve backward compatability

### CODE REVIEW

File: 0021-versioned-data-in-purl-and-stacks.md [hunk 1]
```diff
@@ -6,15 +6,23 @@ permalink: records/0021
...
+  * Justin Coyne
+  * Michael Giarlo
+  * Justin Littman
+  * Marlo Longley
+  * Peter Mangiafico
+  * John Martin
+  * Niqui O'Neill
+  * Steve Taylor
+  * Vivian Wong
+  * Laura Wruble
```
Comment (mjgiarlo) [inline 0021-versioned-data-in-purl-and-stacks.md:22]:
```suggestion
  * Laura Wrubel
```

Comment (mjgiarlo) [inline 0021-versioned-data-in-purl-and-stacks.md:22]: This looks like most of the E&D developers + Vivian, but not all of them (3-4 omitted). Intentional?

File: 0021-versioned-data-in-purl-and-stacks.md [hunk 2]
```diff
@@ -6,15 +6,23 @@ permalink: records/0021
...
+* Date(s):
   * Drafted: 2024-03-15
-  * ...
+  * Approved: 2024-06-24
```
Comment (mjgiarlo) [inline 0021-versioned-data-in-purl-and-stacks.md:25]: Trivial tweak but I think we approved it yesterday?

```suggestion
  * Approved: 2024-06-25
```

File: 0021-versioned-data-in-purl-and-stacks.md [hunk 3]
```diff
@@ -74,6 +83,15 @@ Chosen option: "Oxford Common File Layout", because this has the capability of s
...
 * Bad, because multiple versions of each object would show up in discovery and management interfaces.
 
+### Custom file layout with content addressable addressable storage
+
+* Good, because no migration of data or systems is required (using symbolic links)
+* Good, because data is deduplicated between versions
+* Neutral, we have to build and document this approach. We are not sharing with any other institutions.
+* Good, because it runs on common file systems.
+* Good, because it's a relatively simple.
+* Bad, because links have to be kept in synch with content addressed files. (e.g. two sources of truth)
```
Comment (mjgiarlo) [inline 0021-versioned-data-in-purl-and-stacks.md:93]: LGTM

**Review by mjgiarlo — APPROVED** [review]
Approve pending consideration of suggestions

File: 0021-versioned-data-in-purl-and-stacks.md (hunk 1 above)
Comment (jcoyne) [reply 0021-versioned-data-in-purl-and-stacks.md]: This was the list of attendees from the meeting notes.

**Pull request merged**